ui/tract-renderer/src/main/java/org/cru/godtools/tract/viewmodel/CardViewHolder.java

```diff
@@ -176,22 +176,11 @@ private boolean isPrayerForm(Card card) {
         return false;
     }
 
-    private boolean isPrayerSelection(Card card) {
-        if (card != null) {
-            for (Event.Id dismissListener : card.getDismissListeners()) {
-                if (dismissListener.name.contains("followup-form")) {
-                    return true;
-                }
-            }
-        }
-        return false;
-    }
-
     private int getCardCount() {
         int count = 0;
         if (mModel != null) {
             for (Card card : mModel.getPage().getCards()) {
```

Comment:
shouldn't the card count be the total number of visible cards instead of filtering out the arbitrary `PrayerForm` card?

visible cards is a much less brittle number to calculate